upgrade/v18xto190/upgrade.go

```diff
@@ -6,36 +6,45 @@ import (
 	clientset "k8s.io/client-go/kubernetes"
 
 	lhclientset "github.com/longhorn/longhorn-manager/k8s/pkg/client/clientset/versioned"
+	"github.com/longhorn/longhorn-manager/types"
 	upgradeutil "github.com/longhorn/longhorn-manager/upgrade/util"
 )
 
-const (
-	upgradeLogPrefix = "upgrade from v1.8.x to v1.9.0: "
-)
+type resourceMapLoader func(namespace string, lhClient *lhclientset.Clientset, resourceMaps map[string]interface{}) error
```

Comment:
The name of the type is a bit weird and confusing and is unable easily realize what it does.

Suggest
```
listAndUpdateResourcesInProvidedCacheFunc
```